.buildkite/pipeline.full.yml

```diff
@@ -75,6 +75,91 @@ steps:
           - maze_output/metrics.csv
     commands:
       - features/scripts/run-macos-ci-tests.sh
+
+  - label: ':windows: Build windows test fixture for Unity 2020'
+    timeout_in_minutes: 30
+    key: build-windows-fixture-2020
+    depends_on: build-artifacts
+    agents:
+      queue: windows-unity-wsl
+    env:
+      UNITY_PERFORMANCE_VERSION: 2020.3.48f1
```

Comment:
Maybe not for now, but can we use something like [this](https://stackoverflow.com/questions/48940619/yaml-how-to-reuse-single-string-content) so that we only define the absolute version of Unity once for each major and use throughout the config file to avoid one test having a slightly different version?

Reply:
I managed to alias everything and make it so that you can set the exact unity version in one place at the top of the file.